modules/client/examples/06-multisig-client/05-can-approve.ts

```diff
@@ -0,0 +1,23 @@
+/* MARKDOWN
+### Checking if user can approve in a multisig plugin
+*/
+import {
+  MultisigClient,
+  Context,
+  ContextPlugin,
+  ICanVoteParams,
+} from "@aragon/sdk-client";
+import { contextParams } from "../00-client/00-context";
+
+// Create a simple context
+const context: Context = new Context(contextParams);
+// Create a plugin context from the simple context
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+// Create an multisig client
+const client = new MultisigClient(contextPlugin);
+
+const canVote = await client.methods.canApprove("0x1234567890123456789012345678901234567890");
```

Comment:
Don't we need to pass the pluginAddress and the proposal ID? If you already approved a certain proposal you would receive `false`

Reply:
Didn't think of that, I saw it more of a plugin wide thing, but maybe `isMember` would be a better name for what I was thinking of. For this makes sense passing the proposalId. In fact with the new proposalId definition we wont even need the plugin address, just the proposalId